### PR TITLE
IBX-963: Buttons on Trash page are hidden on medium resolutions

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-963
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

https://github.com/ezsystems/ezplatform-page-builder/pull/827

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

File: src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig [hunk 1]
```diff
@@ -6,148 +6,150 @@
 {% set custom_actions = custom_actions|default(false) %}
 {% set have_actions = (is_draft or is_archived) or custom_actions %}
 
-<table class="ibexa-table table ibexa-table__draft-conflict {{ (is_draft and have_to_paginate ? 'mb-3' : '') }}">
-    <thead>
-        <tr class="ibexa-table__head-row">
-            {% if form is defined %}
+<div class="ibexa-scrollable-wrapper">
+    <table class="ibexa-table table ibexa-table__draft-conflict {{ (is_draft and have_to_paginate ? 'mb-3' : '') }}">
```
Comment (lucasOsti) [inline src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig:10]: you don't have an `ibexa-table` parent so i think it should either be `ibexa-draft-conflict` or `ibexa-table--draft-conflict`

File: src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig [hunk 2]
```diff
@@ -6,148 +6,150 @@
...
-                    'locationId': location.id ?? null
-                })
-            %}
+                {% block custom_actions_column_headers %}{% endblock %}
+                {% if have_actions %}
+                    {% include '@ezdesign/ui/component/table/table_head_cell.html.twig' with { content: null } %}
+                {% endif %}
+            </tr>
+        </thead>
+        <tbody
```
Comment (lucasOsti) [inline src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig:44]: maybe one line?:)

File: src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig [hunk 3]
```diff
@@ -6,148 +6,150 @@
...
+                                    {% endif %}
+                                {% endblock %}
+                            {% endembed %}
+                        {% endif %}
+                    {% endblock %}
+                {% endembed %}
+            {% endfor %}
+        </tbody>
+    </table>
+</div>
```
Comment (lucasOsti) [inline src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig:155]: new line

**Review by lucasOsti — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/views/themes/admin/ui/component/table/table.html.twig [hunk 4]
```diff
@@ -13,47 +13,49 @@
...
-                        head_cols: head_cols,
-                    } %}
-                {% endif %}
-            {% endblock %}
-        </thead>
-        <tbody
-            class="ibexa-table__body
-            {{ table_body_class|default('') -}}"
-            {{ html.attributes(table_body_attr|default({})) }}
+    <div class="ibexa-scrollable-wrapper">
```
Comment (tischsoic) [inline src/bundle/Resources/views/themes/admin/ui/component/table/table.html.twig:16]: Do we want to make it an option?

Comment (tischsoic) [reply src/bundle/Resources/views/themes/admin/ui/component/table/table.html.twig:16]: I think you didn't understand me. I was thinking about adding an option for 3rd party to disable with default value `true`.

**Review by lucasOsti — APPROVED** [review]

**Review by tischsoic — APPROVED** [review]

**Pull request merged**